jest.config.js

```diff
@@ -24,6 +24,7 @@ module.exports = {
             transform: {
                 '^.+\\.jsx?$': `<rootDir>/jest-preprocess.js`,
             },
+            transformIgnorePatterns: ['/node_modules/(?!(@leafygreen-ui)/)'],
```

Comment:
Was running into an issue where jest could not compile the leafygreen Code component (also updated) as-is. Transforming it (in testing only) remedies this issue.

![Screen Shot 2020-05-27 at 3 10 45 PM](https://user-images.githubusercontent.com/9064401/83062196-42539300-a02c-11ea-8619-08bdeafd8ec8.png)